docs/testcoverage.rst

```diff
@@ -0,0 +1,86 @@
+Test Coverage
+=============
+
+You can integrate Coverage.py with behave-django to find out the test coverage of your code.
+
+Dependencies
+------------
+
+At First, you should install Coverage.py dependency
+
+.. code-block:: bash
+
+    $ pip install coverage
+
+Environment.py
+--------------
+
+In ``environment.py``, add the code snippet below in the ``before_all`` function to start measuring test coverage:
+
+.. code-block:: python
+
+    import coverage
+
+    def before_all(context):
+        cov = coverage.Coverage()
+        cov.start()
+        context.cov = cov
```

Comment:
This looks like code that we could add [to our management command](https://github.com/behave/behave-django/blob/main/behave_django/management/commands/behave.py#L151-L153), doesn't it? – The feature could be activated by a `--coverage` command line switch.